namesrv/src/main/java/org/apache/rocketmq/namesrv/NamesrvStartup.java

```diff
@@ -121,7 +121,10 @@ public static NamesrvController createNamesrvController(String[] args) throws IO
         log = InternalLoggerFactory.getLogger(LoggerName.NAMESRV_LOGGER_NAME);
 
         MixAll.printObjectProperties(log, namesrvConfig);
```

Comment:
This has already print the config in logs, you can check the configs in namesrv_default.log

Reply:
oh，okay， good job